.github/workflows/neuron-ci.yml

```diff
@@ -238,8 +238,7 @@ jobs:
                                 -Wno-unused-but-set-variable \
                                 -Wno-reorder \
                                 -Wno-sign-compare \
-                                -Wno-strict-aliasing \
-                                -Wno-mismatched-tags" \
+                                -Wno-strict-aliasing" \
                              -DNRN_EXTRA_MECH_CXX_FLAGS="-Wno-sometimes-uninitialized \
```

Comment:
```suggestion
                             -DNRN_EXTRA_MECH_CXX_FLAGS="-Wno-sometimes-uninitialized -Wno-mismatched-tags\
```
?

Reply:
`-Wno-strict-aliasing` doesn't seem to be needed in the current clang-14 build, I wonder if it was a hangover from an older iteration with GCC (or maybe the underlying issue was fixed already).

Not sure about the second suggestion, do you just want different whitespace?

Reply:
that's ok